mediator/med_phases_restart_mod.F90

```diff
@@ -14,6 +14,9 @@ module med_phases_restart_mod
   use med_phases_prep_glc_mod , only : FBocnAccum2glc_o, ocnAccum2glc_cnt
   use med_phases_prep_rof_mod , only : FBlndAccum2rof_l, lndAccum2rof_cnt
   use pio                     , only : file_desc_t
+#ifndef CESMCOUPLED
+  use shr_is_restart_fh_mod, only : init_is_restart_fh, is_restart_fh, is_restart_fh_type
```

Comment:
This module is missing - did you git add it?

Reply:
It should be found in our CDEPS share directory (ufs/cdeps_share) which sits at the UFS-WM CDEPS interface level. These are where we have files these files

```
dtypes.h
shr_assert_mod.F90
shr_frz_mod.F90
shr_infnan_mod.F90
```

which are generated via genf90.pl for you (and which we cannot use because of restrictions by CentralOps).

Reply:
Looking at directory cdeps/share I see files shr_assert_mod.F90.in, shr_frz_mod.F90.in and shr_infnan_mod.F90.in but no shr_is_restart_fh_mod.F90.in
or shr_is_restart_fh_mod.F90 - that file needs to go in that directory if only to enable github testing of the ufs build.

Reply:
So a ``shr_is_restart_fh_mod.F90`` inside of cdeps/share will be enough? It doesn't need to be a ".in" file, right?